src/Stratis.Benchmark/Delegates.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using DBreeze;
+using NBitcoin;
+using Stratis.Bitcoin.Base.AsyncWork;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Benchmark
+{
+    public class DelegateTests
```

Comment:
What does this do?

Reply:
was to test the performance difference between passing a method ref (new delegate) and between passing Func, but I can remove this

Reply:
ok so yeah delete it